sub-projects/device/src/AWSClass.py

```diff
@@ -0,0 +1,45 @@
+import boto3
+import json
+
+#this class is for telescreen_device activity
+class AWS(object):
+	
+	#initalize the session for AWS
+	def __init__(self):
+		self.session = boto3.Session(profile_name='default')
+	
+	#uploading img to S3
+	def uploadS3(self, img):
```

Comment:
This method and ones that follow may need doc strings (instead of the inline comments above).

See https://google.github.io/styleguide/pyguide.html#Comments under "Functions and Methods". 

I will let you decide what meets the criteria in the above link required for not having a doc string.